dashboard/views.py

```diff
@@ -224,6 +232,14 @@ def file_access_within_week(request, course_id=0):
             else:
                 output_df=output_df.drop([i_grade], axis=1)
 
+
+    if (file_type != "all_files"):
+        # drop all other files
+        file_types = [CANVAS_FILE, LECCAP_FILE]
```

Comment:
again, you need to consider how to make the file type list dynamic. We do not want to hard code the file type here

Reply:
I will try to fix this